<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MUZIK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MUZIK</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pBtnPrev">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Previous</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/icon/previous.png</normaloff>../resource/icon/previous.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+Left</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnPlay">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Play</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/icon/play.png</normaloff>../resource/icon/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Space</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnNext">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Next</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/icon/next.png</normaloff>../resource/icon/next.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+Right</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnPlayMode">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Order</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/icon/order.png</normaloff>../resource/icon/order.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+Right</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelNow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDuration">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderPlayProgress">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>590</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderVolume">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>18</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnVolume">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Mute On/Off</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/icon/volume.png</normaloff>../resource/icon/volume.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnPlaylist">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Playlist</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resource/icon/list.png</normaloff>../resource/icon/list.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
